README.md

```diff
@@ -1,65 +1,65 @@
 
-You can learn more about Hardware Observer on [Charmhub](https://charmhub.io/hardware-observer).
-## Getting Started
+Prometheus Hardware Exporter is recommended for [Juju](https://juju.is/) users,
+you can deploy Prometheus Hardware Exporter using the Hardware Observer charm.
+You can learn more about Hardware Observer on
+[Charmhub](https://charmhub.io/hardware-observer).
 
-Install the snap from snap store:
+**Note**: this exporter does not bundle the required third party or proprietary
+software. If you would like to use this exporter, you will need to install them
+by yourself.
 
-```bash
-$ sudo snap install prometheus-hardware-exporter --classic
-```
-
-Start the exporter:
-
-```bash
-$ sudo snap start prometheus-hardware-exporter
-```
-
-## Snap Configuration
+## Installation
 
-The install hook (`./snap/hooks/install`) will generate a default configuration
-for the exporter. By default, the exporter is started at port 10000 with a
-logging level of INFO.
+This package is not published anywhere as of now. Currently, it can be installed
+locally or on a virtual environment. To install it on a virtual environment,
+run:
 
-You can change the default configuration by editing
+```shell
+# Install virtualvenv and source it
+pip3 install virtualenv
+python3 -m venv venv
+source venv/bin/activate
 
-```bash
-$ /var/snap/prometheus-hardware-exporter/current/config.yaml
+# Install the prometheus-hardware-exporter package; the "(venv)" prefix on the
+# shell prompt indicates that the virtual environment is active.
+(venv) pip3 install .
 ```
 
-and then restart the snap by
+To verify that the package has been installed successfully, run:
 
-```bash
-$ sudo snap restart prometheus-hardware-exporter
+```shell
+(venv) prometheus-hardware-exporter -h
 ```
 
-## Local Build and Testing
+If you see the help message, then `prometheus-hardware-exporter` is installed on
+the virtual environment.
 
-You need `snapcraft` to build the snap:
+## Usages
 
-```bash
-$ sudo snap install snapcraft --classic
-```
+To start the exporter at port `10000` and enable the ipmi sensor collector, run:
 
-Snapcraft also requires backend to create isolated build environment, you can
-choose the following two backends:
+```shell
+(venv) prometheus-hardware-exporter -p 10000 --collector.ipmi_sensor
+```
 
-- [LXD](https://linuxcontainers.org/lxd/introduction/), which creates container
-  image build instances. It can be used inside virtual machines.
-- [Multipass](https://multipass.run/), which creates virtual machine build
-  instances. It cannot be reliably used on platforms that do not support nested
-  virtualization. For instance, Multipass will most likely not run inside a
-  virtual machine itself.
+Note that all the exporters are disabled by default as you will need to install
+the appropriate third party or proprietary software to run the collectors.  You
+can check out [Resources](200~https://charmhub.io/hardware-observer/resources/)
```

Comment:
The link seems incorrect. There is a "200- " in front of the http link.

Reply:
Fixed :)